pkg/testutils/dir.go

```diff
@@ -21,7 +21,18 @@ import (
 // TempDir creates a directory and a function to clean it up at the end of the
 // test.
 func TempDir(t testing.TB) (string, func()) {
-	dir, err := ioutil.TempDir("", fileutil.EscapeFilename(t.Name()))
+	tmpDir := ""
+	if RunningUnderBazel() {
+		// Bazel sets up private temp directories for each test.
+		// Normally, this private temp directory will be cleaned up automatically.
```

Comment:
Previously we returned a no-op cleanup function, but that's changed now. I assume that's what we want?